packages/calling/src/CallingClient/CallingClient.test.ts

```diff
@@ -153,11 +156,12 @@ describe('CallingClient Tests', () => {
     it('CALLING: verify invalid service domain', async () => {
```

Comment:
I feel some of the test cases here might not be required, let's discuss the ones which needs to be removed

Reply:
Yeah removed as per our discussion.